s3_file_uploads/utils.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
+from rest_framework.exceptions import ValidationError
+
+def validate_acl_type(data):
+    if data and data.get('acl') not in settings.ACCESS_CONTROL_TYPES:
+        raise ValidationError('{acl} is not a valid acl type'.format(acl=data.get('acl')))
+    return data
+
+
+def get_file_data_from_request_data(request_data):
```

Comment:
Could be useful to provide a type for `request_data` here, to prevent `request_data.items()` from blowing up

Reply:
function removed